File: final/server/src/main/java/com/google/maps/gaming/zoinkies/models/Item.java
package com.google.maps.gaming.zoinkies.models;

/**
 * This class models an item in the game.
 */
public class Item {

  private String ObjectType;
  private int Quantity;

  public Item() {
  }

  public Item(String ObjectType, int Quantity) {
    this.ObjectType = ObjectType;
    this.Quantity = Quantity;
  }

  // region Getters/Setters
  public String getId() {
    return ObjectType;
  }
  public void setId(String ObjectType) {
    this.ObjectType = ObjectType;
  }
  public int getQuantity() {
    return Quantity;
  }
  public void setQuantity(int Quantity) {
    this.Quantity = Quantity;
  }
  // endregion
  @Override
  public String toString() {
    return "{Type: " + ObjectType + " Quantity: " + Quantity + "}";
  }
}

File: part1/client/Zoinkies/Assets/Zoinkies/Scripts/Models/Item.cs
/**
 * Models a game item. Items can be anything, from a character type to a spawn location
 * to a reward item.
 */
public class Item {
    public string id { get; set; }
    public int quantity { get; set; }

    public Item() {}

    public Item(string ObjectType, int Quantity) {
        this.id = ObjectType;
        this.quantity = Quantity;
    }
    public override string ToString() {
        return "{Type: " + id + " Quantity: " + quantity + "}";
    }
}
